/rm_decision_node:
  ros__parameters:
    route1: [0.5, 1.6, 0.0,
              0.5, 2.4, 0.0,
             -1.0, 2.4, 0.0]
    route2: [2.4, 0.0, 0.0,
             3.7, 0.0, 0.0]
    route3: [4.475, -1.65, 0.0,
             4.475, -2.65, 0.0]
    route4: [2.4, 0.0, 0.0,
             3.7, 0.0, 0.0]        
    home_pose: [0.0, 0.0, 0.0]
